.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,52 @@
+name: Build and Push Backstage Docker Image
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: The version to build and push
+        required: true
+        type: string
+  repository_dispatch:
+    types: [backstage_release]
+
+jobs:
+  docker-build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20.x'
+
+      - name: Install Dependencies
+        run: yarn install --immutable
+
+      - name: Build Backend
+        run: |
+          yarn tsc:full
+          yarn build:all
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
```

Comment:
Why do you need buildx?
you build in `amd64` for `amd64`

Reply:
can you elaborate on that? how do we know the runner architecture? is that a default?

Reply:
dropped it since the default runner is amd64
https://github.com/env0/env0-backstage-plugin/pull/81/commits/046db6e83dd790416cf7fb80742828302ffa7f1f

Reply:
We know what we are running with, and what github uses to run the actions with